### DISCUSSION TITLE
I want to give a fixed temperature value to the heater in OMEdit.

### DISCUSSION THREAD

**Discussion by ZackeyZAKI** [OpenModelica/OpenModelica#10705] [Q&A] [answered]
Please let me know.

To make the outlet temperature of the heater reach 30°C (fixed value) in OMEdit, you need to connect something to the "T" input.

![image](https://github.com/OpenModelica/OpenModelica/assets/131832405/d1109ad6-d52a-4228-bfcd-ba34c70f701f)

**Comment by casella** [comment]
I'm sorry, but I don't understand the question. Can you show the Modelica code of that heater model? That would help 😅

**Reply by ZackeyZAKI** [reply]
Thank you for your response. 

Here is the relevant portion of "hea" of the code.

  Buildings.Fluid.HeatExchangers.Heater_T hea(redeclare package Medium = Buildings.Media.Water, QMax_flow = -300, T_start(displayUnit = "K"), allowFlowReversal = true) annotation(
    Placement(visible = true, transformation(origin = {40, 4}, extent = {{7, -8}, {-7, 8}}, rotation = 0)));
equation
  connect(hea.port_b, volume.ports[1]) annotation(
    Line(points = {{33, 4}, {20.5, 4}, {20.5, 12}, {8, 12}}, color = {0, 127, 255}));

I would be happy to review it.

**Reply by ZackeyZAKI** [reply]
I would like to set the outlet temperature for a heater in OMEdit. 
Because, the comment for "Buildings.Fluid.HeatExchangers.Heater_T" states that it is a "Heater with prescribed outlet temperature." 
However, I cannot find where to set the outlet temperature for Heater_T.

**Comment by casella** [comment, marked as answer]
As you wrote, you need to connect some signal to the T input. You can take them from Modelica.Blocks.Sources

**Reply by ZackeyZAKI** [reply]
Thank you!
When I connected a 'realExpression' at the 'T' point, I was able to define it.
